src/celengine/astrocat.h

```diff
@@ -0,0 +1,68 @@
+
+#pragma once
+
+#include <climits>
+#include <string>
+#include <celutil/util.h>
+#include <fmt/printf.h>
+
+class AstroCatalog
+{
+public:
```

Comment:
one space before `public/private`

Reply:
Should or shouldn't be? And space for sure, not new line, for example?

Reply:
One space should be before any access specifier, see coding-standards.html